backend/app.py

```diff
@@ -43,8 +43,9 @@ def product():
 
     product_data = {}
     source_details = {}
-    if "Best Buy" in source:
+    if "BestBuy" in source:
```

Comment:
What's the reason the space is being removed? From a frontend standpoint, having the space is much simpler.

Reply:
I didn't realize the endpoint URL format had changed so it wasn't working for me initially. But I changed it back now.

Reply:
Ahh okok, that's probably my bad. I must've changed something without notifying you